packages/stainless/src/stl.ts

```diff
@@ -315,11 +459,115 @@ type OpenAPITopLevel<
   ? {}
   : { actions: { getOpenapi: OpenAPIEndpoint } };
 
+/** Parameters for {@link Stl.endpoint}. */
+interface EndpointParams<
+  MethodAndUrl extends HttpEndpoint,
+  Config extends EndpointConfig | undefined,
+  Path extends ZodObjectSchema | undefined,
+  Query extends ZodObjectSchema | undefined,
+  Body extends ZodObjectSchema | undefined,
+  Response extends z.ZodTypeAny = z.ZodVoid
+> {
+  /**
+   * a string declaring the HTTP method
+   * and URL for this endpoint
```

Comment:
```suggestion
   * A string declaring the HTTP method
   * and path for this endpoint, eg:
   * "get /items/{id}"
```

Reply:
btw I've found it awkward that the term "endpoint" is overloaded to refer to either the whole object or just the HTTP method and URL associated with it.  In a cursory google search it seems like most people take endpoint to mean a URL, but not necessarily an associated HTTP method.

Reply:
Agreed... any ideas?

Reply:
MethodAndPath is the only unambiguous term I can think of unfortunately

Reply:
One idea I had recently was `definition` as in "endpoint definition"…

Reply:
Could also do "route"?

Reply:
Yeah, hmmm... the only other word that comes to mind rn is "callsite"